api/src/main/java/org/pmiops/workbench/dataset/DataSetMapper.java

```diff
@@ -3,14 +3,29 @@
 import org.mapstruct.Mapper;
 import org.mapstruct.Mapping;
 import org.pmiops.workbench.db.model.DbDataDictionaryEntry;
+import org.pmiops.workbench.db.model.DbDataset;
+import org.pmiops.workbench.db.model.DbStorageEnums;
 import org.pmiops.workbench.model.DataDictionaryEntry;
+import org.pmiops.workbench.model.DataSet;
+import org.pmiops.workbench.model.PrePackagedConceptSetEnum;
 import org.pmiops.workbench.utils.mappers.CommonMappers;
 
 @Mapper(
     componentModel = "spring",
     uses = {CommonMappers.class})
 public interface DataSetMapper {
 
+  @Mapping(target = "id", source = "dataSetId")
+  @Mapping(target = "conceptSets", ignore = true) // This is stored as a list of ids. Look those up in the controller for entities
```

Comment:
Since the mapper is a Spring bean itself, you can write a little method to do these mappings using the real DAO. 

When I use a mapper, I expect everything that can be mapped to be mapped.

Reply:
That is a behavior change from our current mapper, and I actually disagree in this case, because it would require more database lookups. I am okay making it a dataSetLight if we want though. The controller lookup for these objects is something @als364 recommended while we were pairing.